app/Controllers/categoryController.php

```diff
@@ -41,18 +41,18 @@ public function createAction() {
 			invalidateHttpCache();
 
 			$cat_name = trim(Minz_Request::param('new-category', ''));
-			if ($cat_name == '') {
+			if ($cat_name === '') {
```

Comment:
That is the only line that is OK, and I would rather not change such a minor thing except when rewriting this piece of code for another reason (e.g. feature, bug fix) or if really required for a task such as https://github.com/FreshRSS/FreshRSS/issues/4112